client/landing/stepper/declarative-flow/flows/onboarding/onboarding.ts

```diff
@@ -123,7 +124,7 @@ const onboarding: FlowV2 = {
 
 			if ( isMvpOnboarding ) {
 				return [
-					addQueryArgs( `/overview/${ providedDependencies.siteSlug }`, { ref: flowName } ),
+					addQueryArgs( `/home/${ providedDependencies.siteSlug }`, { ref: flowName } ),
```

Comment:
Instead of `/hosting` we want all users to land in the same place they currently land, which is the /home page.